start.py

```diff
@@ -715,6 +734,43 @@ def _configure_kudu(deployment, cluster, kudu_version):
             # Only one parcel can be activated at a time, so once one is found, our work is done.
             break
 
+
+def _validate_kudu_version(kudu_version, cdh_version, cm_cluster):
+    cdh_version_tuple = tuple([int(i) for i in cdh_version.split('.')])
```

Comment:
Nit: pretty sure you don't need the `[` and `]` inside `tuple`

Reply:
Done.